src/search/search_engine.cc

```diff
@@ -173,11 +173,11 @@ void SearchEngine::add_succ_order_options(OptionParser &parser) {
 }
 
 void print_initial_evaluator_values(
-    const EvaluationContext &eval_context, utils::LogProxy &log) {
+    const EvaluationContext &eval_context) {
```

Comment:
With the decision to have an if-guard at the lowest level (where prints actually happen), we should probably re-insert handing the logger to this function and also to the ones it calles that use the logger as well.

Reply:
I don't understand how this is related. Now each Evaluator has its own log, so we don't need to hand it in anymore. Instead, if search wants to modify logging of its evaluators, then the evaluators should be created with the right option. (I agree that it feels more natural to use the logger of the search, but implementing hierarchical loggers is for the future.)